go.mod

```diff
@@ -44,7 +44,6 @@ require (
 	github.com/h2non/filetype v1.0.12
 	github.com/hashicorp/go-azure-helpers v0.10.0
 	github.com/hashicorp/go-plugin v1.0.1
-	github.com/hashicorp/go-retryablehttp v0.6.4 // indirect
```

Comment:
Why did this get removed? Isn't it required by the terraform provider?

Reply:
Looks suspicious. Must have fat fingered something last night. Will investigate.

Reply:
go mod vendor|verify keeps removing this for some reason. Can't figure out why? When I force add it, the CI job for vendoring fails.

Reply:
I assume because it's no longer an indirect dependency, but now a direct one?  I'm not sure why there's not an explicit entry for it.

Reply:
Maybe because terraform-provider-ironic's go.mod brings it in